### ISSUE TITLE
Can't generate PnL Report

### ISSUE THREAD

**Issue by PabloCastellano** [rotki/rotki#7009]
## Problem Definition

In rotki 1.31.0 there are 2 issues that I wasn't experiencing before.

When I click on the `Profit and Loss Report` tab an error notification is shown

![image](https://github.com/rotki/rotki/assets/73274/99c60140-13da-45d2-9c2b-e80987f348b0)

Also the report won't end successfully 

> An unexpected response was received while generating the profit and loss report. This might indicate an issue with the generation mechanism. Please check the logs for any error messages (you might have to enable debug logging first) and create an issue.
>
> The backend query task died unexpectedly: **'ValidatorDailyStats' object has no attribute 'event_identifier'**

![image](https://github.com/rotki/rotki/assets/73274/91365ec7-ef36-4353-a06d-e57a4756da98)

## Logs

Clicking Profit and Loss Report tab:

```
2023-11-27T13:02:36.066Z: (error): [
  {
    "code": "invalid_type",
    "expected": "boolean",
    "received": "undefined",
    "path": [
      "entries",
      0,
      "settings",
      "includeFeesInCostBasis"
    ],
    "message": "Required"
  }
]
```

When report it about to end

```
[27/11/2023 14:09:33 CET] ERROR rotkehlchen.api.rest Greenlet with id 140627884780992: Greenlet for task 52 dies with exception: 'ValidatorDailyStats' object has no attribute 'event_identifier'.
Exception Name: <class 'AttributeError'>
Exception Info: 'ValidatorDailyStats' object has no attribute 'event_identifier'
Traceback:
   File "src/gevent/greenlet.py", line 908, in gevent._gevent_cgreenlet.Greenlet.run
  File "rotkehlchen/api/rest.py", line 413, in _do_query_async
  File "rotkehlchen/api/rest.py", line 1528, in process_history
  File "rotkehlchen/rotkehlchen.py", line 847, in process_history
  File "rotkehlchen/accounting/accountant.py", line 164, in process_history
  File "rotkehlchen/accounting/accountant.py", line 304, in _process_event
  File "rotkehlchen/accounting/structures/evm_event.py", line 289, in process
  File "rotkehlchen/accounting/history_base_entries.py", line 108, in process
```

### System Description

Operating system: Ubuntu
Rotki version: 1.31.0

**Comment by LefterisJP** [comment]
Okay so these are indeed 2 different issues.

### Frontend zod error

 The first one with the zod error is something that @lukicenturi is looking at and has been reported in the discord.

The "hack" for now is to delete the transient DB for your user. This is the `rotkehlchen_transient.db` which contains old reports

### Backend logs

This I haven't seen before. Will need to investigate.

**Comment by LefterisJP** [comment]
The log you provided was really helpful @PabloCastellano!

Figured it out and made a PR to include the fix in the next patch release: https://github.com/rotki/rotki/pull/7010

**Comment by LefterisJP** [comment]
Should be fixed by #7010
